app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -338,6 +340,10 @@ private IdentifierHolder saveParticipant(Study study, Set<Roles> callerRoles, St
         if (study.isExternalIdValidationEnabled()) {
             options.remove(EXTERNAL_IDENTIFIER);
         }
+        // You can't update time zone after it is set.
```

Comment:
So if the participant goes on a trip, or moves to a different time zone, or heaven forbid their local jurisdiction switches between standard and daylight time, is the time zone now wrong? Do we care?